README.md

```diff
@@ -13,10 +13,23 @@ You need the following things installed if you want to run directly on the host
 
 ```bash
 ./enter.sh # only if you are running using the Docker environment
-cd XXX
+cd <which_physics>
 ./get_data.sh
-make
-./main.x
+cd ../runfile
+export BACKEND=<backend>
+python physics.py <which_physics> <data_dir> <backend> <which_tile> <which_savepoint>
+```
+
+For example, to validate microphysics using `gtx86` backend for all tiles and savepoints:
+```
+export BACKEND=gtx86
+python physics.py microph ../microph/data $BACKEND None None
```

Comment:
So I wanted the capability to input file path because I could just point the path to the saved data on daint, but when running this in docker, it should just be "../" + which_physics + "/python" inside physics. Making this an optional input now